judge/admin/vote.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+
+
+class VoteAdmin(admin.ModelAdmin):
+    list_display = ('points', 'voter', 'problem', 'note')
+    search_fields = ('voter', 'problem')
+
+    # if the user has edit all problem or other editing permissions, so superusers
+    def has_change_permission(self, request, obj=None):
+        if obj is None:
+            return request.user.has_perm('judge.edit_own_problem')
+        return obj.problem.is_editable_by(request.user)
+
+    def lookup_allowed(self, key, value):
+        return super().lookup_allowed(key, value) or key in ('problem__code',)
```

Comment:
```suggestion
        return super().lookup_allowed(key, value) or key in ['problem__code']
```
:bike: :house: ing

Reply:
Tuples are preferred over lists.

Reply:
So should I make the change or?

Reply:
No. @Ninjaclasher just wondering, why? (feel free to explain elsewhere)

Reply:
See https://github.com/DMOJ/online-judge/pull/895#discussion_r261494479.